src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -432,9 +432,18 @@ private void writeVcfTuple(final VcfTuple tuple, final VariantContextWriter writ
             final List<Allele> truthAlleles = alleles.truthAlleles();
             final List<Allele> callAlleles  = alleles.callAlleles();
 
+            // Since they differ in the no call case, use the site (rather than the genotype) alleles for new VC
```

Comment:
I think this comment is more relevant to this PR than to the code itself, how about:
```
// Get the alleles present at this site for both samples to use for the output variant context.
```
?